CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 2.40.7 / 2022-12-13
+
+* [BUGFIX] Use Windows native DNS resolver. #11704
+* [BUGFIX] TSDB: Fix positive buckets used instead of negative buckets. #11699
```

Comment:
Let's flag more clearly here that this only affects the experimental native histograms.

How about: "TSDB: Fix queries involving negative buckets of native histograms."